ckanext/unaids/dataset_transfer/logic.py

```diff
@@ -0,0 +1,108 @@
+import logging
+
+from ckan import model
+import ckan.plugins.toolkit as toolkit
+from ckan.lib import mailer
+from ckan.lib.base import render_jinja2
+from ckan.common import config, _
+from ckanext.unaids.dataset_transfer.model import (
+    DatasetTransferRequest,
+    STATUS_EMAILED,
+    STATUS_EMAIL_FAILED
+)
+
+log = logging.getLogger(__name__)
+
+
+def get_org_admins_with_email_addresses(org, exclude_user_ids):
+    try:
+        org_admin_ids = [
+            user['id']
+            for user in org['users']
+            if user['capacity'] == 'admin'
+        ]
+    except KeyError:
+        raise ValueError('{} {} {}'.format(
+            _('Organization'),
+            org['name'],
+            _('has no users to accept this transfer')
```

Comment:
The message for this ValueError shouldn't be translated. It's just internal implementation and shouldnt' surface to the UI.
Also the message is a little misleading. The function is about getting admins from an organization. There's no transfer of ownership in this context. The message should indicate how to use it properly and what went wrong, e.g.
"Provided organization {org.get(id)} doesn't include information about users. organization('users') = {org.get('users')}"

Reply:
Yeah I wasn't sure about this as there isn't much consistency as to if we translate error messages 😅 
I'll change it as advised